docs/docker-setup.md

```diff
@@ -1,28 +1,43 @@
 # Dev Testing Set Up Notes
 
-## Prerequisites
+## Prerequisites:
 
-1. [Download](https://docs.docker.com/compose/install/) and set up Docker for your machine
-  * [New Start Up Guide](https://www.docker.com/get-started)
-  * Optionally, directly install the latest version of [docker-compose](https://github.com/docker/compose)
+1. Docker: [Download](https://docs.docker.com/compose/install/) and set up Docker for your machine
+   - [New Start Up Guide](https://www.docker.com/get-started)
```

Comment:
Feel free to adjust this too, but I'd probs do it quickly in a diff PR to keep it succinct